src/pages/slots/SlotConfiguration.tsx

```diff
@@ -0,0 +1,72 @@
+import { MainContainer, Label, Input, Button, Select, InputContainer, EditContainer, EditCapacity, TitleContainer, Title, ScreenContainer } from "./SlotConfiguration.styles";
+import EditIcon from "../../assets/edit-icon.png";
+import EditSlotForm from "./forms/EditCapacityForm";
+import { useRef, useState } from "react";
+import AddIcon from "../../assets/add-icon.svg";
+import Modal from "../../components/modal/Modal";
+import { DaysOfWeek } from "../../utils/DaysOfWeek";
+
+function CreateSlot() {
+
+    const [showModal, setShowModal] = useState<boolean>(false);
+
```

Comment:
Para que el botón Nuevo Turno se habilite al seleccionar un día, habria que manejar el estado del Select
```suggestion
const [selectedDay, setSelectedDay] = useState("");
```